src/client/app/components/LoginComponent.tsx

```diff
@@ -2,12 +2,26 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import React from 'react';
+import * as React from 'react';
 import axios from 'axios';
 import { browserHistory } from 'react-router';
 import HeaderComponent from '../components/HeaderComponent';
 
-export default class LoginComponent extends React.Component {
+interface NotificationSpec {
+	message: string, level: string, position: string, autoDismiss: number
```

Comment:
Better to have these on multiple lines.
There also might be additional parameters that can be passed to ReactNotificationSystem.